src/main/java/ch/njol/skript/Skript.java

```diff
@@ -21,6 +21,7 @@
 import ch.njol.skript.aliases.Aliases;
 import ch.njol.skript.bukkitutil.BurgerHelper;
 import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.util.MarkedForRemoval;
```

Comment:
I can't seem to find the use for this import

Reply:
ah yup you're right completely forgot about that from when I was testing